### ISSUE TITLE
Could'nt find Qt Project File 

### ISSUE THREAD

**Issue by Aurentiaco35** [jusirkka/qopencpn#7]
In project file I could not find the .pro file. Are you using QT IDE for this project . Can you share that file if there is a chance?  Is there any way to zoom in on Linux version. I tried the mouse wheel and control combinations that does not work.

**Comment by jusirkka** [comment]
It's a cmake based build. Qt-creator supports cmake projects: Open Project -> navigate to qutenav dir -> click on CMakeLists.txt. The zoom in/out uses standard key bindings, Ctrl+ and Ctrl- at least in my case.

**Comment by Aurentiaco35** [comment]
Yes thank you. I found it but when I run the app I couldn't see ant general map information doesn't have any map in it. I find a way to add chart but it just shows a piece but there is not a map included in application.

**Comment by Aurentiaco35** [comment]
I think I have some problems with databases but I don't understand how can I add the file that you suggested I download it and insert it on chart part on the app but it does not working what should I do can you help me about it?

**Comment by Aurentiaco35** [comment]
And if there is a chance can I get your email address or discord or any social media account to communicate with you.

**Comment by jusirkka** [comment]
I'll write more detailed chart installation instructions later, I don't have time to do that today.

**Comment by Aurentiaco35** [comment]
Thank you. I am waiting your instructions. Thank you again

**Comment by Aurentiaco35** [comment]
Hi again, could you find time to write about the instructions. How can I connect gps to qutenav. Could you explain it to me if there is a chance ?

**Comment by dogualpay** [comment]
Hello, I have the same problem with @Aurentiaco35 I run the application and see that map does not showing, grayscreen that I see.  I couldn't see informations about database or the file upload.  Are you planning to share a documentation for chart installiation ? @jusirkka
